kroxylicious-sample/README.md

```diff
@@ -8,27 +8,31 @@ This sample filter project provides examples to help you learn how [custom filte
 
 Building the sample project is easy! You can build the **kroxylicious-sample** jar either on its own or with the rest of the Kroxylicious project.
 
-To build all of Kroxylicious, including the sample:
+#### To build all of Kroxylicious, including the sample:
 
 ```
-$ mvn clean install
+$ mvn verify --also-make
 ```
 
-To build the sample on its own:
+#### To build the sample on its own:
 
 ```
-$ mvn clean install -pl kroxylicious-sample
+$ mvn verify -pl :kroxylicious-sample --also-make
```

Comment:
I think this is the only place we actually need to use `also-make`.

The other usages we are running the full build